WordPress/src/main/java/org/wordpress/android/ui/reader/discover/interests/ReaderInterestsViewModel.kt

```diff
@@ -168,10 +168,10 @@ class ReaderInterestsViewModel @Inject constructor(
 
     private fun transformToInterestsUiState(interests: ReaderTagList) =
```

Comment:
Perhaps change the reference/methods names so that it no longer reflects the deleted `InterestUiState` class? Not needed, but may be easier for future us to understand later. wdyt?

Reply:
I have to rename these variables and classes to avoid tag/topic/interest confusion. Will do in a separate PR. Added note here: https://github.com/orgs/wordpress-mobile/projects/63#card-43573240